packages/core/src/StakePoolSearchProvider/types/StakePool.ts

```diff
@@ -0,0 +1,119 @@
+import { Hash16, PoolParameters, PoolMetadata } from '@cardano-ogmios/schema';
```

Comment:
I'd like to see the `StakePool` type not bounded to the search provider, as it's more of a general concept in Cardano